src/main/resources/application.yml

```diff
@@ -89,7 +89,9 @@ logging:
   pattern:
     console: "%clr(%d{dd-MM-yyyy HH:mm:ss.SSS}){faint} %clr(${LOG_LEVEL_PATTERN:-%5p}) %clr([%35.35t]){faint} %clr(%-28.28logger{28}){cyan} %clr(:){faint}%X{BUSINESS-LOG} %m%n${LOG_EXCEPTION_CONVERSION_WORD:-%wEx}"
 
+springdoc:
+  writer-with-order-by-keys: true
+
 gsma:
   payee:
-    tenant: "lion"
-
+    tenant: lion
```

Comment:
Is this a sensible default?

Reply:
anyways we are going to override it is respective chart based on environment. So I think its fine.